qdrant-landing/content/documentation/data-ingestion-beginners.md

```diff
@@ -113,87 +113,35 @@ docs = loader.load()
 
 To get things rolling, we’ll use two powerful models:
 
-1. **OpenAI Embeddings** for transforming text data.
-2. **CLIP (Contrastive Language-Image Pretraining)** for image data.
-
-### Text Embeddings
-
-Here, we’re using `OpenAIEmbeddings`—a pre-trained model that turns text into embeddings, capturing its underlying meaning. With these, we’re ready to unlock powerful search and retrieval tasks.
-
-Here’s how to set up the OpenAI embedding model for text:
-
-```jsx
-from langchain_openai import OpenAIEmbeddings
-
-# Initialize the text embedding model from OpenAI
-text_embedding_model = OpenAIEmbeddings(model="text-embedding-3-small")
-```
+1. **`sentence-transformers/all-MiniLM-L6-v2` Embeddings** for transforming text data.
+2. **`CLIP` (Contrastive Language-Image Pretraining)** for image data.
 
 ---
 
-The `text-embedding-3-small` model generates high-quality text embeddings, making it easier to find documents with similar meanings—even if they don’t contain the exact same words.
-
-### Image Embeddings
-
-We’re using OpenAI’s `ClipModel`, which is designed to handle both images and text. Here, we’ll use CLIP to generate embeddings for images, allowing you to compare image content based on its semantic meaning.
-
-Here’s how to set up the CLIP model and processor::
-
-```jsx
-From transformers import CLIPProcessor, CLIPModel
-import torch
-
-# Initialize the CLIP model and processor
-clip_model = CLIPModel.from_pretrained("openai/clip-vit-base-patch32")
-clip_processor = CLIPProcessor.from_pretrained("openai/clip-vit-base-patch32")
-```
+### Document Processing Function
 
----
+![data-ingestion-beginners-8.png](/documentation/examples/data-ingestion-beginners/data-ingestion-8.png)
 
-The `clip-vit-base-patch32` model is specifically trained to align images and text within the same embedding space, meaning that similar images will have embeddings that are close to each other. With the models set up, you can now create a function to process documents based on their type and generate embeddings using the CLIP model, for instance, to convert image features into vectors.
+Next, we’ll create a `process_document` function that converts various file types—such as text, PDF, and image—into embeddings. This function applies different methods to extract and embed content based on the file type, making it versatile for handling multiple formats effectively.
```

Comment:
This description does not match what we do
We're not creating `process_document` anymore, and we're not computing embeddings here